### DISCUSSION TITLE
Question about installing PHP extension by passing PHP_CONFIGURE_OPTIONS

### DISCUSSION THREAD

**Discussion by CSY54** [asdf-community/asdf-php#114] [Q&A]
I wished to build PHP with ldap support (`--with-ldap`), so I used `PHP_CONFIGURE_OPTIONS="--with-ldap" asdf php install 8.0.14`.

However, after building it, it seems that `curl` couldn't get HTTPS resources, turns out that OpenSSL wasn't being included during configuring PHP.

After tracing `install` script, options from `$(os_based_configure_options)` and `$PHP_CONFIGURE_OPTIONS` are mutually exclusive, which makes `PHP_CONFIGURE_OPTIONS="--with-ldap"` overwrite the *default* flags such as `--with-openssl --with-curl --with-zlib --with-readline --with-gettext` generated by `os_based_configure_options`. ([install#L156-L160](https://github.com/asdf-community/asdf-php/blob/master/bin/install#L156-L160))

Is this the intended behavior of how `PHP_CONFIGURE_OPTIONS` works? or `PHP_CONFIGURE_OPTIONS` should allow us to add different flags from `os_based_configure_options`?

**Comment by redesigned** [comment]
Did you try appending to the options instead of overwriting them?:
```
export PHP_CONFIGURE_OPTIONS="$PHP_CONFIGURE_OPTIONS  --with-ldap"
```
